README.md

```diff
@@ -28,9 +28,21 @@ require('skywalking-nodejs').start({
 });
 ```
 
+### Download and deploy skywalking backend.
+It's very important that nodejs agent and Skywalking backbend that your download should be compatible. Please see [the document](docs/capatibility-list.md) and Skywalking provide two deploy mode, one is [standalone mode](https://github.com/apache/incubator-skywalking/blob/master/docs/en/Deploy-backend-in-standalone-mode.md) and the other is [cluster mode](https://github.com/apache/incubator-skywalking/blob/master/docs/en/Deploy-backend-in-cluster-mode.md)
+
+### Reboot application
```

Comment:
Also already included in the document. Please remove.

Reply:
The `reboot application`  is part of the deployment step.  I think adding this section will make more sense.